site/content/notifications/givenow.md

```diff
@@ -0,0 +1,7 @@
+---
+id: "1617046894617923400"
+title: "Givenow"
+date: 2021-03-29T15:41:34-04:00
+headless: true
+---
+##### **Consider [donating to OpenCourseWare](https://giving.mit.edu/give/to/ocw/?utm_source=ocw&utm_medium=homepage_banner&utm_campaign=nextgen_home) to help further our efforts.**
```

Comment:
We should use the text from https://docs.google.com/document/d/1FR3-IOhB2iSkQNR7QK7YIpL5ip4GVV3fFiHWGiWOQQU/edit

How will this work to link to the page in #71?

Reply:
@pdpinch This was just temporary, as I was waiting for that PR to merge but it has now.  Linking to the other page would be as simple as adding a generic page with the markdown content in the doc, then linking to it with an absolute link.  Update incoming on that.

Reply:
Although I should have indicated that with a comment...